routes/web.php

```diff
@@ -66,6 +66,8 @@
     Route::get('/', 'OverviewController@index')->name('home');
     Route::view('vminfo', 'vminfo');
 
+    Route::any('nac', 'NacController@index');
```

Comment:
```suggestion
    Route::get('nac', 'NacController@index');
```

Only need the get verb I think.